app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,110 @@
+#update_email_btn {
+	display: none;
+}
+
+#remove_account_btn {
+	width: 65%;
+	color: #4E6472;
+	background-color: #102F44;
+	border: none;
+}
+
+#add_number_btn {
+	background: transparent;
+	color: white;
+	text-shadow: 0 0 5px white;
+	width: 65%;
+	border-color: white;
+	box-shadow: 0 0 8px white;
+	position: relative;
+	border: 2px solid #fff;
+	box-shadow: 0px 0px 10px #fff;
+}
+
+#add_number_btn::after {
+	content: "";
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: -1;
+	box-shadow: inset 0px 0px 10px #fff;
+}
+
+#remove_number_button {
+	color: red;
+	background: none;
+	border: none;
+}
+
+#confirm_btn, #ok_btn, #submit_code_btn {
+	border: 2px solid white ;
+	box-sizing: border-box;
+	background-color: white;
+	color: #102F44;
+	padding: 0 20px;
+}
+
+#ok_btn {
+	margin-left: 50%;
+}
+
+#submit_code_btn {
+	margin-left: 50%;
+}
+
+#user_email {
+	width: 45%;
+	height: 100%;
+	margin: 25px;
+	padding: 3px 10px;
+	border: 1px solid  #dddddd;
+}
+
+#user_email_style {
+	label {
+		border: 2px solid white ;
+		box-sizing: border-box;
+		background-color: white;
+		color: #102F44;
+		padding: 0 20px;
+		abbr {
+			display: none;
+		}
+	}
+}
+
+ul {
+  list-style-type: none;
+}
+
+#new_user_phone {
+	padding-bottom: 20px;
+	/*width: 25%;*/
+	label {
+		abbr {
+			display: none;
+		}
+	}	
+}
+
+/*#user_phone_form {
+	width: 25%;
+	margin: 0 auto;
+	padding-bottom: 20px;
+}*/
+
+#user_phone_number {
+	width: 25%;
+	border: 1px solid  #dddddd;
+}
+
+#user_phone_code {
+	width: 25%;
+	border: 1px solid  #dddddd;
+}
+
+#user_phones {
+	padding: 0;
+}
```

Comment:
I have done some style for inputs and buttons in main.scss , it would be cool you use them to keep the same style for all pages(just use my classes). and add margins&paddings, pleas , similar to pattern (check design issue).